pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -43,6 +43,10 @@ func (ds *dockerService) getSecurityOpts(seccompProfile string, separator rune)
 	return nil, nil
 }
 
+func (ds *dockerService) getSandBoxSecurityOpts(separator rune) []string {
+	return nil
```

Comment:
Potentially add a comment with the reason for not returning `no-new-privileges`? - Would potentially be helpful for future folks.

Reply:
Thanks for your note, comment added